settings.py

```diff
@@ -1643,6 +1643,7 @@
     'corehq.messaging.smsbackends.start_enterprise.models.StartEnterpriseBackend',
     'corehq.messaging.smsbackends.ivory_coast_mtn.models.IvoryCoastMTNBackend',
     'corehq.messaging.smsbackends.airtel_tcl.models.AirtelTCLBackend',
+    'corehq.messaging.smsbackends.test.models.SQLTestSMSWithAPIBackend',
```

Comment:
Is this for tests only? If yes, would it make sense to add it in `testsettings.py` instead?

Reply:
Yes. It felt strange putting it here (was following the `SQLTestSMSBackend` class) so I will make that change.